switchmate.py

```diff
@@ -16,10 +17,13 @@
 import sys
 import ctypes
 
+from time import time
```

Comment:
@brianpeiris You can remove this library import now since your code doesn't use it. I don't want to create a pull request because it will overwrite my forked code.